.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -135,9 +135,38 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      # We always build the image but we only push if we are on the `main`,
-      # `master` branch or a versioned `v*` branch
-      - name: Build and Push Docker Image (amd64 and arm64)
+      # Build amd64 images always, and publish when it is not a fork. The Github
+      # security model prevents forks from pushing to the registry so we can
+      # only push if the branch/PR is not generated from a fork. Even though
+      # forks can't push, we still want to try and build the image to catch
+      # bugs. For testing purposes we only need an amd64 image.
+      - name: Build and Push Docker Image amd64
+        uses: docker/build-push-action@v4
+        env:
+          OUTPUT_SHORT_SHA: ${{ needs.prepare-env.outputs.output_short_sha }}
+          OUTPUT_IMAGE_NAME: ${{ needs.prepare-env.outputs.output_image_name }}
+        with:
+          context: .
```

Comment:
@jrmanes I added this here because you added it in the other step and I assume we should have it here as well.

Reply:
> @jrmanes I added this here because you added it in the other step and I assume we should have it here as well.

yes please @MSevey , this should fix the issue with the ldflags in Docker